sirbot/pythondev/slack/do.py

```diff
@@ -0,0 +1,17 @@
+DIGITAL_OCEAN_REFFERRAL_URL = 'https://m.do.co/c/457f0988c477'
+DIGITAL_OCEAN_URL = 'https://digitalocean.com'
+
+DIGITAL_OCEAN_TEXT = '''Here at Python Developers we host our website and
+Slack bot on <{}|Digital Ocean>. If you are planning on using Digital Ocean,
+please use our <{}|referral code>. You get 10 USD and help support the
+ community!'''.format(DIGITAL_OCEAN_URL, DIGITAL_OCEAN_REFFERRAL_URL)
+
+
+def add_to_slack(slack):
+    slack.add_command('/do', func=share_digital_ocean, public=False)
```

Comment:
public should be set as `True` here because we want the response to be seen by everyone.

When setting public as `False` the response the message is sent as `ephemeral` so only the user triggering the command can see it. Another way is to set `response.response_type = 'in_channel'`